### PR TITLE
feat: use cash.min.js on the page instead of CDN

### PR DESCRIPTION
**Pull request by raisedadead**
Syncs changes from https://github.com/freeCodeCamp/auth0-templates/pull/18

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

File: pages/login.html [hunk 1]
```diff
@@ -437,7 +430,6 @@ <h3> Thanks for your patience. </h3>
               </form>
             </div>
           </div> <!-- End passwordless-email -->
-          <br />
```
Comment (ojeytonwilliams) [inline pages/login.html:440]: I doubt this matters much, but removing it seems unrelated to the rest of the PR.

Comment (raisedadead) [reply pages/login.html:440]: I was copy/pasting stuff from the org tenant, so could be that.
